### ISSUE TITLE
DB 7.2.0.2020008302047..  mySQL column "comment" and then SAVE

### ISSUE THREAD

**Issue by SherlockSoftware** [dbeaver/dbeaver#9703]
#### System information: 
- WIndows 10
- DBeaver 7.2.0.2020008302047

#### Connection specification:
- mySQL       com.mysql.jdbc.authentication.MysqlNativePasswordPlugin

#### Describe the problem you're observing:
Change to Comment column the SAVE button still disabled.

### How to...
select a Table tab / Properties / Comment field/column visable
Change  comment text.. the SAVE button is still disabled..

Work around change something in another column and back.. 
The Save button now works to select PERSIST.

Phil

**Comment by uslss** [comment]
Hi @SherlockSoftware 
cannot reproduce it.
What driver do you use? Could you post a video that shows issue?
Thanks in advance

**Comment by SherlockSoftware** [comment]
Howdy///

This is going to be the EFC [ edit Focus change ] where in the text below “casse” and me taking and image shot triggers the SAVE button to ENABLE.
Before I selected the image program the SAVE button was DISABLED.

If I type  “aaa” in caaase the SAVE button does not ENABLE [ does in this picture… ] until I click on another control.  Focus change.
Even though I have changed the contents of the field.. it is only focus change the says something changed.

Would not matter what driver I would imagine it is the Windows controls.

“Edit Change” or “Edit Focus Change” events.
.
You must only use the later, which is fine you know.

Hope this helps..


[cid:image001.png@01D68567.D5D378A0]

[cid:image002.png@01D68567.D5D378A0]
